docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -150,7 +150,8 @@ The table below maps the GitHub Cloud and GitHub Enterprise roles to the corresp
 
 <sup>1</sup>: Outside collaborators aren't supported as members of organizations on Codacy. You can still [add outside collaborators to Codacy](managing-people.md#adding-people) so that Codacy analyzes their commits to private repositories, but they won't be able to join your Codacy organization.  
 <sup>2</sup>: Joining an organization may need an approval depending on your setting for [accepting new people](changing-your-plan-and-billing.md#allowing-new-people-to-join-your-organization).  
-<sup>3</sup>: Requires that an organization owner has given the Codacy GitHub App access to the repositories to add or remove.
+<sup>3</sup>: These users can only see security items originating from Jira and from Codacy repositories that they follow.  
```

Comment:
we need to double-check this: I believe that as a consequence of only showing issues for repositories that users follow, that for Organization Members, we won't show Jira items at all

Reply:
Thanks @mrfyda! Keeping this on hold for now.

Reply:
yes, just confirmed with @afsmeira:
members won't ever see Jira items
Organization manager and Admins will see them when not filtering for repositories